eng/testing/runsettings.targets

```diff
@@ -46,6 +47,12 @@
                       WriteOnlyWhenDifferent="true"
                       Overwrite="true" />
     
+    <WriteLinesToFile File="$(VsCodeRunSettingsOutputFilePath)"
```

Comment:
Did you consider Copy task here instead?   Doesn't make much difference since this is a small amount of work but might make the intent clearer.

Reply:
I didn't consider copy, is there a preference of copy over `WriteLinesToFile`?